Shared/Prefs.swift

```diff
@@ -74,7 +77,6 @@ public struct PrefsKeys {
     public static let AppExtensionTelemetryOpenUrl = "AppExtensionTelemetryOpenUrl"
     public static let AppExtensionTelemetryEventArray = "AppExtensionTelemetryEvents"
     public static let KeyBlockPopups = "blockPopups"
-    public static let KeyShowSponsoredShortcuts = "showSponsoredShortcuts"
```

Comment:
We're removing this key and renaming it above.... was it used anywhere before? Do we have to worry about users that already have it?

Reply:
Sponsored shortcuts wasn't released yet, so I think this is fine